spec/lib/jobs_spec.rb

```diff
@@ -88,6 +88,13 @@
       end
     end
 
+    context 'when the search phrase includes "federal" or "gobierno" ' do
+      it 'should not trigger job search' do
+        %w['federal' 'gobierno'].
+          each { |phrase| expect(Jobs.query_eligible?(phrase)).to be_falsey }
```

Comment:
As we really only need to test that generic, non-job-related words are not query eligible, I would simplify this to:
```ruby
expect(Jobs.query_eligible?('federal')).to be_falsey
```

Reply:
I put both because in the future if a dev plans to take it out the test will fail and they will then see that we wanted those two phrases not to be used. Otherwise someone may think it was must an over site.

Reply:
I wouldn't worry about that, because someone would need to intentionally add those words back in for some reason. Otherwise, we could argue that we should test any number of words that someone might add as a keyword. Testing two words rather than one makes the spec less clear, because we only need to verify that unrelated words are not query eligible, rather than specific words.